azure/pipeline/update_trans_container.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t transform_h5_container -f ./pipeline/transform_h5_container/Dockerfile .
+docker tag transform_h5_container:latest 351672045885.dkr.ecr.us-west-2.amazonaws.com/transform_h5_container:latest
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 351672045885.dkr.ecr.us-west-2.amazonaws.com
+docker push 351672045885.dkr.ecr.us-west-2.amazonaws.com/transform_h5_container:latest
```

Comment:
The docker image released to dockerhub does not relay on `azure` module, right? If not, then can exclude `azure` directory in `.dockerignore`, so to keep docker image size small.

Reply:
The Docker image does require some files from the azure module. If you open the Dockerfile, you will see that I only copy exactly what I need, so the image is already as small as possible.

I don't think an update to .dockerignore is needed, but let me know if I'm misunderstanding something!